SPECS/systemd/systemd.spec

```diff
@@ -1174,6 +1179,7 @@ fi
 %if 0%{?want_bootloader}
 %files ukify -f .file-list-ukify
 %files boot -f .file-list-boot
+/boot/efi/EFI/BOOT/grubx64.efi
```

Comment:
This is the same location that grub will install its efi binary in. Likely you will need a package Conflicts here on the grub-efi package

Reply:
Updated